packages/flutter/lib/src/cupertino/refresh.dart

```diff
@@ -359,43 +357,53 @@ class CupertinoSliverRefreshControl extends StatefulWidget {
     return state.refreshState;
   }
 
-  /// Builds a simple refresh indicator that fades in a bottom aligned down
-  /// arrow before the refresh is triggered, a [CupertinoActivityIndicator]
-  /// during the refresh and fades the [CupertinoActivityIndicator] away when
-  /// the refresh is done.
-  static Widget buildSimpleRefreshIndicator(
+  /// Builds a refresh indicator that reflects the standard iOS pull-to-refresh
+  /// behaviour. Specifically, this entails presenting an activity indicator that
+  /// changes depending on the current refreshState. As the user initially drags
+  /// down, the indicator will gradually reveal individual ticks until the refresh
+  /// becomes armed. At this point, the animated activity indicator will begin rotating.
+  /// Once the refresh has completed, the activity indicator shrinks away as the
+  /// space allocation animated back to closed.
+  static Widget buildAppleRefreshIndicator(
```

Comment:
Why the name change? I think we avoid using 'Apple' explicitly in the framework.

Reply:
The old name was `buildSimpleRefreshIndicator` which, to me, was accurate in that it was a simple implementation that didn't deal with the complexities of how iOS renders its indicator. Having the full implementation labelled "simple" didn't feel quite right.  Your point is taken about referencing Apple, though, so I've renamed it to simply `buildRefreshIndicator`